src/main/java/Parser.java

```diff
@@ -0,0 +1,257 @@
+				System.out.println("Now you have " + counter + " tasks in the list.\n" + ui.BORDER);
+				wf.appendToFile("E/" + newEvent.getStatus() + "/" +
+						newEvent.getDescription() + "/" + newEvent.getVenue() + "\n");
+
+			} catch (DukeInvalidArgumentException e) {
+
+				System.out.println(ui.BORDER + "\n" + e + "\n" + ui.BORDER);
+
+			} catch (DukeException e) {
+
+				System.out.println(ui.BORDER + "\n" + e + "\n" + ui.BORDER);
+			} catch (IOException e) {
+
+				System.out.println(ui.BORDER + "\n" + e + "\n" + ui.BORDER);
+			}
+
+		} else if (input.startsWith("todo")) {
+			try {
+				if (input.length() < 5) {
+					throw new DukeException(ui.EMPTY_INPUT);
+				}
+
+				String taskToDo = input.substring(5);
+				Todo newToDo = new Todo(taskToDo);
+				tasks.addTask(newToDo);
+				counter = tasks.getCounter();
+				System.out.println(ui.BORDER + "\nGot it. I've added this task:");
+				System.out.println(newToDo.toString());
+				System.out.println("Now you have " + counter + " tasks in the list.\n" + ui.BORDER);
+				wf.appendToFile("T/" + newToDo.getStatus() + "/" +
+						newToDo.getDescription() + "\n");
+
+			} catch (DukeException e) {
+				System.out.println(ui.BORDER + "\n" + e + "\n" + ui.BORDER);
+			} catch (IOException e) {
+
+				System.out.println(ui.BORDER + "\n" + e + "\n" + ui.BORDER);
+			}
+
+		} else if (input.startsWith("delete")) {
+			try {
+				String[] taskDelete = input.split(" ");
+				if (taskDelete.length == 1) {
+					throw new DukeException("☹ OOPS!!! Which task would you like to delete?");
+				}
+
+				if (taskDelete.length > 2) {
+					throw new DukeInvalidArgumentException(ui.DELETE_FORMAT, input);
+				}
+
+				int taskIndex = Integer.parseInt(taskDelete[1]);
+				String removedTask = tasks.getTask(taskIndex - 1).toString();
+				tasks.deleteTask(taskIndex - 1);
+				System.out.println(ui.BORDER + "\nNoted. I've removed this task:");
+				System.out.println(removedTask);
+
+				counter = tasks.getCounter();
+				System.out.println("Now you have " + counter + " tasks in the list.\n" + ui.BORDER);
+				wf.writeToFile(Duke.writeFile(tasks.getTaskList()));
+
+			} catch (DukeInvalidArgumentException e) {
+
+				System.out.println(ui.BORDER + "\n" + e + "\n" + ui.BORDER);
+
+			} catch (NumberFormatException e) {
+
+				System.out.println(ui.BORDER + "\nOOPS!! Please input a task number!\n" + ui.BORDER);
+
+			} catch (IndexOutOfBoundsException e) {
+
+				System.out.println(ui.BORDER + "\nTask number not found! Try again!\n" + ui.BORDER);
+
+			} catch (DukeException e) {
+
+				System.out.println(ui.BORDER + "\n" + e + "\n" + ui.BORDER);
+
+			} catch (IOException e) {
+
+				System.out.println(ui.BORDER + "\n" + e + "\n" + ui.BORDER);
+
+			}
+		} else {
+			try {
+				throw new DukeException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+			} catch (DukeException e) {
+				System.out.println(ui.BORDER + "\n" + e + "\n" + ui.BORDER);
+			}
+		}
+	}
```

Comment:
It'd be nice if this method did solely parsing and the execution and text output were done elsewhere.